lib/internal/cli_table.js

```diff
@@ -11,8 +11,7 @@ const { getStringWidth } = require('internal/util/inspect');
 // The use of Unicode characters below is the only non-comment use of non-ASCII
 // Unicode characters in Node.js built-in modules. If they are ever removed or
 // rewritten with \u escapes, then a test will need to be (re-)added to Node.js
-// core to verify that Unicode characters work in built-ins. Otherwise,
-// consumers using Unicode in _third_party_main.js will run into problems.
+// core to verify that Unicode characters work in built-ins.
```

Comment:
Fwiw, I would like to keep this comment, because I think it’s still worth testing that this works for scripts which are passed through our `js2c` converter. For the embedding use case specifically, https://github.com/nodejs/node/pull/33972 should cover this.